Library/Homebrew/formula_creator.rb

```diff
@@ -166,6 +168,12 @@ def install
             bin.env_script_all_files(libexec/"bin", :PERL5LIB => ENV["PERL5LIB"])
         <% elsif mode == :python %>
             virtualenv_install_with_resources
+        <% elsif mode == :ruby %>
+            ENV["GEM_HOME"] = libexec
+            system "gem", "build", "\#{name}.gemspec"
+            system "gem", "install", "\#{name}-\#{version}.gem"
```

Comment:
I'm wondering if we should/could try to use `bundle install --standalone` here instead. That would avoid needing to `resource` all the dependent gems, I think?

Reply:
All dependencies are being installed by `gem install` AFAIK. Couldn't find any example of a formula in core that uses `bundle install`.

Reply:
Yeh, fair enough, that seems fine 👍